css/home/showcase.css

```diff
@@ -34,20 +44,29 @@
   /* justify-self: flex-end; */
 }
 
-.showcase-form .form-control {
+.showcase__form .card {
+  width: 400px;
+}
+
+.showcase__form--control {
   margin: 30px 0;
 }
 
-.showcase-form input[type="text"],
-.showcase-form input[type="email"] {
+.showcase__form--hidden {
+  visibility: hidden;
+  height: 0;
+}
+
+.showcase__form input[type="text"],
+.showcase__form input[type="email"] {
   border: 0;
   border-bottom: 1px solid #b4becb;
   width: 100%;
   padding: 3px;
   font-size: 16px;
 }
 
-.showcase-form input:focus {
+.showcase__form input:focus {
```

Comment:
Same as previous input,  target input with BEM class.

Reply:
no puedo replicar el .showcase-form , no existe

Reply:
If showcase.css is not part of home, please remove from the home directory or delete the unneccessary css for the future.